plugins/scaffolder-backend/sample-templates/all-templates.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
```

Comment:
I'm wondering if it makes sense to push these examples to the `examples` option when creating the action and then people get them for free everywhere in the actions view rather than having to dig around for this template? :pray:

Reply:
Hey, yes, those are in the examples available here https://github.com/drodil/backstage/blob/6fc03c7d3ab0b1ff75ea037c113314bf47fecb99/plugins/scaffolder-backend-module-notifications/src/actions/sendNotification.examples.ts#L20

Adding this template mainly allows easier debugging of notifications and allows them to be seen in action when running the local development environment. Especially at the moment, where none of the core features yet use notifications. Of course, the notifications can be done with `curl` or a similar direct API call but it's a bit of a hassle every time.